src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -211,6 +214,10 @@ export class TabsTitleControl extends TitleControl {
 			}
 		}));
 
+		// Configuration updates
```

Comment:
Can we try to put this into the [`ITabOptions`](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/services/group/common/groupService.ts#L22) bag and use the related [`IEditorGroupsService.onTabOptionsChanged`](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/services/group/common/groupService.ts#L65) event?

Reply:
I looked into this briefly, and I don't know how to wire it all up. Sorry.

Reply:
ok leave it as is and I will have a look after you are done 👍